src/components/XAxis/index.tsx

```diff
@@ -11,7 +11,7 @@ import { Domain, Series } from '../../external';
 import { DomainsByItemId } from '../Scaler';
 import { withDisplayName } from '../../utils/displayName';
 
-export interface Props {
+export interface Props extends ScalerProps, SizeProps {
```

Comment:
I think `react-sizeme` actually exposes the necessary types

Reply:
I don't know if I want components to take in a `size` prop -- that's a pretty clunky API. I think I'd rather just declare our own size prop, honestly.

Reply:
Ah, well that's what we have currently